lib/collision/AABB.js

```diff
@@ -39,6 +39,8 @@ function AABB(lower, upper) {
   }
   if (typeof upper === 'object') {
     this.upperBound.set(upper);
+  } else if (typeof lower === 'object') {
+    this.upperBound.set(lower);
```

Comment:
Does this require a change in the TS definitions? It seems to me that actually all parameters are optional.